components/eam/bld/build-namelist

```diff
@@ -3104,6 +3115,13 @@ if ($chem eq 'superfast_mam4_resus_mom_soag' || $chem eq 'superfast_mam5_resus_m
     if (! $megan_emis ) {
       %species = ( %species,
 		'ISOP   -> ' => 'isop_emis_file',  );
+      if ($vbs_soa) {
+         %species = ( %species,
+                     'ISOP_VBS -> ' => 'isop_vbs_emis_file',  
+                     'C10H16 -> ' => 'c10h16_emis_file',  
+                     'SOAG0  -> ' => 'soag0_emis_file'  
+                    );
+      }
```

Comment:
@mingxuanwupnnl , is it appropriate to use the $vbs_soa flag to control the addition of the above three species?

Reply:
Yes. That looks good to me.